tests/testsuite.at

```diff
@@ -306,7 +304,7 @@ m4_define([DEFAULT_CATEGORIES], [[Syntactic],
 
 dnl Store user-defined souffle flag configuration given by the SOUFFLE_CONFS env (if any)
 m4_define([ENV_CATEGORIES],
-    m4_split(m4_esyscmd_s(echo "[[$SOUFFLE_CATEGORY]]"),[,]))
+    m4_split(m4_esyscmd_s(echo "[$SOUFFLE_CATEGORY]"),[,]))
```

Comment:
This change effectively disables all categories after the first. On Travis, when it is given a list of categories like `SOUFFLE_CATEGORY=Unit,Syntactic,Semantic,Interface,Profile,Provenance`, only the Unit testing is done.